bundles/org.openhab.binding.solax/src/main/java/org/openhab/binding/solax/internal/SolaxBindingConstants.java

```diff
@@ -61,6 +61,10 @@ public class SolaxBindingConstants {
     public static final String CHANNEL_INVERTER_OUTPUT_FREQUENCY_PHASE1 = "inverter-frequency-phase1";
     public static final String CHANNEL_INVERTER_OUTPUT_FREQUENCY_PHASE2 = "inverter-frequency-phase2";
     public static final String CHANNEL_INVERTER_OUTPUT_FREQUENCY_PHASE3 = "inverter-frequency-phase3";
+    public static final String CHANNEL_INVERTER_TEMPERATURE1 = "inverter-temperature1";
```

Comment:
Any clue what temperature 1 and 2 means? Could be nice if we can somehow make it more explicit if we know it. If we don't know it, I guess it's OK. :) (same applies for the metadata files)